engine/engine_test.go

```diff
@@ -4911,6 +4911,16 @@ func roundValues(r *promql.Result) {
 	}
 }
 
+// dropAnnotations drops anntoations from the result
+// TODO: this should be done in a comparer and is only
+// because prometheus got more liberal with its usage of
+// annotations for info purposes. It should only be used
+// if we use a mock storage that doesnt send the same info
+// annotations.
+func dropAnnotations(r *promql.Result) {
```

Comment:
We should try to mirror the annotations that PromQL adds at some point but this is a larger undertaking so ill just drop them for comparisons sake now. 
It is not optimal and we should follow-up with a PR that does this smarter.